core/pkg/server/handler.go

```diff
@@ -530,23 +530,34 @@ func (h *Handler) handleRunStart(record *service.Record, request *service.RunSta
 		}
 	}
 
+	cwd, err := os.Getwd()
+	var codePathLocal string
+	if err != nil {
+		h.logger.Error("error getting current working directory", "error", err)
+	} else {
+		localPath, err := filepath.Rel(cwd, h.settings.GetProgramAbspath().GetValue())
```

Comment:
is this correct that it will be relative to the current working dir?
could you please explain what is the difference between CodePath and CodePathLocal?

Also do you think we can get the correct path in the client and pass it directly? what is the value of doing the computation here?